connect/api/v1/recent_activity/views.py

```diff
@@ -15,7 +15,23 @@ class RecentActivityAPIView(views.APIView):
 
     def post(self, request):
 
-        if request.data.get("flow_organization"):
+        action = request.data.get("action")
+        entity = request.data.get("entity")
+        entity_name = request.data.get("entity_name")
+        user = User.objects.get(email=request.data.get("user"))
+
+        if(request.data.get("intelligence_id")):
+            organization = Organization.objects.get(inteligence_organization=request.data.get("intelligence_id"))
+            for project in organization.project.all():
+                RecentActivity.objects.create(
+                    action=action,
+                    entity=entity,
+                    user=user,
+                    project=project,
+                    entity_name=entity_name
+                )
+                return Response(status=status.HTTP_200_OK)
```

Comment:
There is a logic error here, as the return is inside a loop, only the first element of the iteration is created!